io.github.dfandrich.gpscorrelate.json

```diff
@@ -0,0 +1,63 @@
+{
+  "app-id":"io.github.dfandrich.gpscorrelate",
+  "runtime":"org.freedesktop.Platform",
+  "runtime-version":"20.08",
+  "sdk":"org.freedesktop.Sdk",
+  "command":"gpscorrelate-gui",
+  "finish-args":[
+    "--socket=fallback-x11",
```

Comment:
You need `"--share=ipc"` as well for x11

Reply:
Done.